### ISSUE TITLE
NotFoundError: IDBDatabase

### ISSUE THREAD

**Issue by JohannaLaenger** [retorquere/zotero-better-bibtex#3069]
### Debug log ID

7QNHQR4P-fio-rt3ZFGXZYVOz

### What happened?

Hi, 
I am getting the error message "NotFoundError: IDBDatabase.transaction: 'metadata' is not a known object store name" when I start my Zotero. 
I found similar bugs in the solved issues in this rep, but all the links to fixes provided there are dead.  Anyways. 
I recently upgraded my Ubuntu (24.04 MATE); since then BetterBibTex does not work: 
Obsidian import notes says BetterBibtex plugin not installed, I can not retrieve metadata for my pdfs anymore; neither from doi nor automatically.  
Upgrading from zotero 6 to zotero 7 (and re-installation) did not resolve the issue. Re-installation of the plugin did not work. 
With that Ubuntu update a few other things on my system broke, so I wonder if it's an issue with BBT or something else. 

Thank you for your help,
J

**Comment by JohannaLaenger** [comment]
Hi retorque, thank you for the swift supply of the test build. 
unfortunately it did not work.  I am still getting the error message on startup. I submitted another log: D283104684
Please let me know if this is a bug beyond BBT. 
I apprechiate your help
J

**Comment by retorquere** [comment]
I can't access `D<number>` logs.

**Comment by JohannaLaenger** [comment]
D4NDGMMB-fio-hR3pezfBgoi8 Here is another log file submitted to BBT. I removed the depreciated plugins in the hope that this would have an impact, but not success.

**Comment by JohannaLaenger** [comment]
Hi retorquere, thank you for being so on top of things, I really appreciate your work :) 
Better Bibtex is working now. Just to keep you updated: While getting the logs I also noticed that my Zotero connector didn't work with Firefox (or with Chrome), this could have also contributed. Anyways, as I mentioned, I had a few more systemic issues, so I ended up just doing a full re-set-up of my linux.  
Thank you again,
J
